common/transport/lwm2m/src/main/java/org/thingsboard/server/transport/lwm2m/server/client/LwM2mClient.java

```diff
@@ -297,11 +297,24 @@ public boolean isResourceMultiInstances(String pathIdVer, LwM2mModelProvider mod
     }
 
     public ObjectModel getObjectModel(String pathIdVer, LwM2mModelProvider modelProvider) {
-        LwM2mPath pathIds = new LwM2mPath(fromVersionedIdToObjectId(pathIdVer));
-        String verSupportedObject = registration.getSupportedObject().get(pathIds.getObjectId());
-        String verRez = getVerFromPathIdVerOrId(pathIdVer);
-        return verRez != null && verRez.equals(verSupportedObject) ? modelProvider.getObjectModel(registration)
-                .getObjectModel(pathIds.getObjectId()) : null;
+        try {
+            LwM2mPath pathIds = new LwM2mPath(fromVersionedIdToObjectId(pathIdVer));
+            String verSupportedObject = registration.getSupportedObject().get(pathIds.getObjectId());
+            String verRez = getVerFromPathIdVerOrId(pathIdVer);
+            return verRez != null && verRez.equals(verSupportedObject) ? modelProvider.getObjectModel(registration)
+                    .getObjectModel(pathIds.getObjectId()) : null;
+        } catch (Exception e) {
+            if (registration == null) {
+                log.error("Failed Registration is null, GetObjectModelRegistration. ", e);
+            }
+            else if (registration.getSupportedObject() == null) {
+                log.error("Failed SupportedObject in Registration == null, GetObjectModelRegistration.", e);
```

Comment:
Please log the "this.endpoint" as well. Otherwise, we don't know which device is affected by the error